api/v1beta2/sparkapplication_types.go

```diff
@@ -640,8 +640,10 @@ type Port struct {
 // MonitoringSpec defines the monitoring specification.
 type MonitoringSpec struct {
 	// ExposeDriverMetrics specifies whether to expose metrics on the driver.
+	//	+optional
```

Comment:
Do they need to be optional? let's fix indent if so.

Reply:
My bad, it was my first intention but i changed my mind, apparently forgot to remove the comment

Reply:
fixed